content/oracles.mdx

```diff
@@ -0,0 +1,139 @@
+
+In essence, there needs to be a "trigger" for the smart contract to perform actions. Oracles typically serve as those triggers when the smart contract requires real world data. Oracles can be someone with a private key for the smart contract, a webhook, a function call, decentralized voting, hardware plugged into the network, or various other solutions that we will discuss below. Each of these solutions have varying guarantees and attributes.
+
+## Oracles
+So, what is an oracle?
+
+Is an oracle RNG for bits? A source of data? Trusted authority? Verifiable authority on the state of the world or a certain claim?
+
+The answer is that practically anything that provides you some answer to the question "What is the state of x" is an oracle, for any given **x**. For example, if you want your smart contract to know what the weather is in Ithaca, New York each day, you could ask someone, check a weather website, call a weather API, or ask a computer hooked up to a thermometer and barometer. All of these solutions are oracles.
+
+
+
+The bigger question that we have here is: what makes a good oracle? To answer that, we need to define the requirements a bit more.
+
+### Why Use Oracles?
+Oracles are the backbone of Smart Contracts
+
+Without them, many smart contracts lose their guarantees
+
+#### Oracles: Example
+
+
+**Online gambling market:**
+- 56,050,000$ annually
+- ~10,000,000 users
+- Roughly 1/20 adults will partake in some form of online gambling
+- 280% Growth since 2009
+
+<Resources
+    title="Gambling Data:" text="This market, with its significant growth and large market cap, has begun to be impacted by Blockchain."
+    resources={[
+        {link: "https://www.statista.com/statistics/270728/market-volume-of-online-gaming-worldwide/", text: "Statista.com"}
+    ]}
+/>
+
+It is typically nearly impossible to reliably determine and verify the results of many things that are used for gambling. For example, a game as simple as blackjack could be rigged in favor of the dealer to the tune of 1% gambling volume per day. With a smart contract, online gambling can be verifiably fair with transparency around any dealer biases present in games.
+
+However, many areas of online gambling are not as simple as blackjack. In sports betting, without an oracle, you cannot determine which team won a game; as a result, sports betting requires some sort of oracle. For a game like the Superbowl, **these contracts may begin to yield tens or hundreds of millions of dollars, creating a massive financial incentive for oracles to behave dishonorably, reporting data that would lead to their bets succeeding.**
+
+At this point, hopefully it is clear why we need oracles: without them, smart contracts are restricted to use cases that solely involve data that already exists on-chain, such as cryptocurrency. Once we have an oracle, we also need to consider if it is reliable enough for the use case at hand.
+
+##### Gambling Legality & Blockchain
```

Comment:
its hard to tell this is a title when reading through, maybe just make it level 4?
```suggestion
#### Gambling Legality & Blockchain
```

Reply:
Sounds good to me - :)